packages/api/src/index.js

```diff
@@ -13,11 +14,12 @@ async function run() {
   const elasticSearchClient = new Client({
     node: config.node
   });
+  fcl.config(config.fcl);
```

Comment:
Fcl related config is fetched here with the other ones. That makes to transit the object through a relative amount of functions though.